roadmap.md

```diff
@@ -0,0 +1,49 @@
+* **Enhanced Excel UIA support:** Improvements to NVDA's UIA (User Interface Automation) support for Microsoft Excel, providing a more robust and intuitive experience for navigating and interacting with spreadsheets.
+* **Support for Microsoft Natural Voices:** Adding support for Microsoft's Natural Voices text-to-speech engine, expanding user choice and providing access to high-quality synthetic voices.
+* **Web front end for add-on store:** Creating a web-based front end for the NVDA Add-on Store, making it more accessible and easier to browse and search for add-ons.
+* **Add-on installation telemetry:**  Gathering anonymous data on add-on installations to understand usage patterns to help prioritise development efforts to improve the add-on ecosystem.
+* **ARM installation telemetry:** Gathering anonymous data on ARM installations, providing insights into changing usage patterns.
+* **Anonymous identifier:** Implementation of a system for generating unique, rotating, anonymous identifiers for NVDA installations, allowing for better data collection and analysis without compromising user privacy.
+* **End of support for windows 8.1, 2012/R2:**  Officially ending support for older versions of Windows, allowing us to focus on developing for modern operating systems and their latest accessibility features.
+
+## Long-Term Priorities
+
+These priorities represent ambitious goals and initiatives that will shape the future of NVDA.
+
+* **Integrated AI image description:** Free, secure, performant, private and VI-focussed AI image description available out of the box.
+* **Secure add-on environment:** Building a more secure add-on ecosystem, protecting users from potentially malicious add-ons.
+* **Magnifier prototype:** Exploring the development of a built-in screen magnification feature as a prototype, to better support our low-vision users.
+* **Feature usage telemetry:**  Implementing anonymous telemetry to track feature usage within NVDA, helping us understand how users interact with the software and identify areas for improvement.
+* **Official training material for NVDA accessibility testers:**  Developing comprehensive training materials for individuals interested in becoming NVDA accessibility testers, ensuring high-quality testing and feedback.
+* **NVDA help AI chatbot prototype:** Exploring the development of an AI-powered chatbot to provide instant support and answer frequently asked questions about NVDA.
+* **Automated C++ and markdown linting:** Implementing automated linting for C++ and Markdown files, further improving code quality and consistency within the project.
+
+
+## Contributing and Feedback
+
+We encourage contributions from the community! Please explore our [GitHub repository](https://github.com/nvaccess/nvda) or [user group](https://nvda.groups.io/g/nvda) on groups.io to learn more about how you can get involved.
```

Comment:
Should this link to the contributing.md page instead of these? 

```suggestion
We encourage contributions from the community!
Please explore our [GitHub repository](https://github.com/nvaccess/nvda) or [user group](https://nvda.groups.io/g/nvda) on groups.io to learn more about how you can get involved.
```